skate-plugin/src/test/kotlin/com/slack/sgp/intellij/featureflags/FeatureFlagAnnotatorTest.kt

```diff
@@ -24,19 +24,40 @@ class FeatureFlagAnnotatorTest : BaseFeatureFlagTest() {
 
   @Test
   fun `test returns empty list when linkify is disabled`() {
-    assertThat(runAnnotator(enabled = false)).isEmpty()
+    assertThat(runAnnotator(enabled = false, "TestFeature.kt", filePattern = "Feature.kt"))
+      .isEmpty()
   }
 
   @Test
   fun `test report symbols when linkify is enabled`() {
-    assertThat(runAnnotator(enabled = true)).isNotEmpty()
+    assertThat(runAnnotator(enabled = true, "TestFeature.kt", filePattern = ".*Feature(s)?\\.kt\$"))
+      .isNotEmpty()
   }
 
-  private fun runAnnotator(enabled: Boolean): List<FeatureFlagSymbol> {
+  @Test
+  fun `test report symbols when comma separated list of regex file patterns`() {
+    assertThat(
+        runAnnotator(enabled = true, "TestFeatures.kt", filePattern = ".*Feature(s)?\\.kt\$")
+      )
+      .isNotEmpty()
+  }
+
+  @Test
+  fun `test return empty list when test file does not match the file pattern`() {
+    assertThat(runAnnotator(enabled = true, "TestFeatures.kt", filePattern = "Feature.kt"))
```

Comment:
Why does this test use a different pattern?

Reply:
Because this test is to check incorrect pattern returns empty list

Reply:
Gotcha. The regex input should escape the period though no?